src/Crowdsale/DynamicPriceSale.sol

```diff
@@ -70,8 +70,10 @@ contract DynamicSale is ContextUpgradeable, ReentrancyGuardUpgradeable, Ownable2
         quote = IERC20Upgradeable(quote_);
         minter = IMinterDAO(minter_);
         currentPrice = 222 ether;
-        saleHardCap = 7000 ether;
+        saleHardCap = 70000 ether;
         treasury = treasury_;
+        priceCurveMinValue = 4109 ether;
+        priceCurveMaxValue = 200000 ether;
```

Comment:
Why set these values here? The values should match what's in the spreadsheet: https://docs.google.com/spreadsheets/d/17PEOgTeItXcquVjt1hvyfUIXu9y0ZHjsAsUpS3DhWFA/edit#gid=1362743190

The idea is that as soon as we deploy the contract, then it becomes the new source of truth for these numbers.